fedimint-cli/src/lib.rs

```diff
@@ -325,7 +325,7 @@ async fn load_or_generate_mnemonic(db: &Database) -> Result<Mnemonic, CliError>
 #[derive(Subcommand, Clone)]
 #[allow(clippy::large_enum_variant)]
 enum Command {
-    /// Print the latest git commit hash this bin. was build with
+    /// Print the latest git commit hash ,this bin was build with.
```

Comment:
The comma is not needed.
```suggestion
    /// Print the latest Git commit hash this bin was built with.
```
Also, the `.` you removed likely symbolised `bin` being an abbreviation; so, it should remain or the abbreviation be expanded.